core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -53,6 +53,8 @@ import { LostChangesModalComponent } from './modal-templates/lost-changes-modal.
 import { AutosaveInfoModalsService } from './services/autosave-info-modals.service';
 import { ChangeListService } from './services/change-list.service';
 import { ExplorationDataService } from './services/exploration-data.service';
+import { ConnectionService, ConnectionState } from 'services/connection-service.service';
+import { ExplorationChange } from 'domain/exploration/exploration-draft.model';
```

Comment:
Are these added in the correct sorted order for imports?

Reply:
Changed the order.